.rultor.yml

```diff
@@ -19,6 +19,9 @@ release:
     git add lib/zold/version.rb
     git commit -m "version set to ${tag}"
     gem build zold.gemspec
+    curl -sSL https://get.rvm.io | bash -s stable --ruby
```

Comment:
@emilianodellacasa what this is for? Ruby is installed in the Docker image already